### ISSUE TITLE
[$10][MSFT-45] In country field tab index value is empty.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2766]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Inspect Element
3. Observe the country field.

**Expected Result**
The tab index value is suppose to accept the positive integers meaning it should accept values like 1 and above , not 0 or -1.

**Actual Result**
A tab index value is not set i.e empty.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%2045

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2766 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2766 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096330 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096330 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096330 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
PR for this issue is : https://github.com/appirio-tech/accounts-app/pull/253

**Comment by tosha5252** [comment]
Tab index is now set

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2766 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097512 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097512 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
@mishacucicea ,  User has set tab index 0, which does not meet the expected result. Please update accordingly. 
Expected : The tab index value is suppose to accept the positive integers meaning it should accept values like 1 and above , not 0 or -1.
 
![image](https://user-images.githubusercontent.com/25297859/62003902-c05db980-b13d-11e9-9a9a-8c5d0cfcac7f.png)

**Comment by mishacucicea** [comment]
@drasticdpk , if the tabindex is 0, it means when using tab key, it will cycle through the items in the order they appear on the page, so the expected functionality is correct. Using tab-key will navigate the user to the country field and it will not be skipped like before.

Positive values for tabindex are used if a special order of items is required (which is not the case here).

As the functionality is working correctly the ticket **can be accepted**.
Note that for future issues, we need to focus on the functionality, not the actual solution. Providing a solution in the ticket expected result is not desired, because there can be multiple solutions for the same issue.

**Comment by ajefts** [comment]
Test in dev passed.

**Comment by nkumar-topcoder** [comment]
@drasticdpk Since no change has been made post prod push and it's reverified in DEV, pls check in Prod and close the ticket. Also, refer to mishacucicea explanation above. Thanks.

**Comment by tosha5252** [comment]
Verified correct....tab-index is set to '0' in prod...which is acceptable

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097512<br/><br/>```This is an automated message for crazyk via Topcoder X```
